src/main/scala/com/ironcorelabs/IronSdk.scala

```diff
@@ -50,10 +50,23 @@ trait IronSdk[F[_]] {
    */
   def groupCreate(options: GroupCreateOpts): F[GroupMetaResult]
 
+  def userCreate(jwt: String, password: String, options: UserCreateOpts): F[UserCreateKeyPair]
+
   /**
    * Accesses advanced SDK operations.
    *
    * @return an instance of the [[IronSdkAdvanced]]
    */
   def advanced: IronSdkAdvanced[F]
 }
+
+object IronSdk {
+  import cats.effect.Sync
+  import cats.implicits._
+  def userCreate[F[_]](jwt: String, password: String, options: UserCreateOpts)(
```

Comment:
Would you add a doc comment to this, please.

Reply:
Added